cmd/vic-init/ops.go

```diff
@@ -43,12 +43,9 @@ type operations struct {
 }
 
 func (t *operations) Setup() error {
-	log.Info("Launching pprof server on port 6060")
-	go func() {
-		log.Info(http.ListenAndServe("127.0.0.1:6060", nil))
-	}()
+	err := pprof.StartPprof("vch-init", pprof.VchInitPort)
```

Comment:
go vet might cry about it so we may very well fix it while the code is in review